camelot/handlers.py

```diff
@@ -22,10 +23,16 @@ class PDFHandler(object):
     pages : str, optional (default: '1')
         Comma-separated page numbers.
         Example: 1,3,4 or 1,4-end.
+    password : str, optional (default: None)
+        Owner or User password required for decryption.
```

Comment:
Should we just change this to "Password for decryption." as you've done with the CLI help, so as to not confuse the user?